### PR TITLE
fixed the merge conflict, please check

### PR DESCRIPTION
**Pull request by Eguchinedu**
fixed the merge conflict, names.txt up todate

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -95,4 +95,5 @@ Sikiru Rashidat
 Patrick Aziken
 Umeh Cassandra
 Medina Folashade Adeniran
-Kingsley Umujeyan
\ No newline at end of file
+Kingsley Umujeyan
```
Comment (onahanthony) [inline names.txt:98]: let me use here and practice comment abeg

Comment (Oluwasetemi) [reply names.txt:98]: Yes

**Comment by Oluwasetemi** [comment]
Please I am sorry for all the trouble we have caused you but count it all joy. Your PR is about to be merged and you will be able to see your name on this [url](https://altschool-opensource-names.oluwasetemi.dev/). 

To enable me to do this, **READ the README again carefully**, update your code with the latest and push the update. I am here for you for the next **6 hours** before I close the PR so you can initiate a fresh PR which will be easier to merge.

**Comment by tobisupreme** [comment]
Hi Chinedu!

Please open a new PR! Remember to [make a branch](https://github.com/Oluwasetemi/altschool-opensource-names#make-a-branch) as you work on your changes! 🙏

We look forward to merging your next PR! :partying_face:

**Comment by tobisupreme** [comment]
No problem. Feel better soon

**Comment by Eguchinedu** [comment]
Thank you for your email, i have created a new PR kindly review.

**Pull request closed without merging**